PopPUNK/utils.py

```diff
@@ -572,11 +573,12 @@ def check_and_set_gpu(use_gpu, gpu_lib, quit_on_fail = False):
 
     # Set memory management for large networks
     if use_gpu:
+        multiprocessing.set_start_method('spawn', force=True)
```

Comment:
Just double checking this is the one that works on all platforms (fork is the problematic one, right?)